extensions/prost/transitive_repositories.bzl

```diff
@@ -1,6 +1,7 @@
 """Definitions for loading transitive `@rules_rust//proto` dependencies"""
 
 load("@bazel_features//:deps.bzl", "bazel_features_deps")
+load("@com_google_protobuf//:protobuf_deps.bzl", "protobuf_deps")
```

Comment:
why does prost require all these protobuf dependencies now? I thought the point of it was that it doesn't?

Reply:
I added this because when I updated protobuf I started getting tons of load errors for random external repositories. If there’s a way to avoid this I’m happy to reduce the amount of forced dependencies!

Reply:
I have no idea sorry.  :(  I wonder how it worked before though?

Reply:
At a glance it seems like protobuf just started adding more dependencies expecting bzlmod to hide that from users 😞

Reply:
So updating and pulling in all those changes that introduced new forced dependencies introduced the need for these extra dependencies.